mappings/net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig.mapping

```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_2906 net/minecraft/world/gen/chunk/OverworldChunkGeneratorConfig
+	FIELD field_13221 minY I
+	FIELD field_13222 forcedBiome I
+	FIELD field_13223 riverSize I
+	FIELD field_13224 biomeSize I
```

Comment:
The getters don't return these (in addition, `getMinY` does not even return the constant value of `field_13221`).

Reply:
so should I just undo those fields?

Reply:
@liach Juuxel is right, the fields aren't used in the getters, they were inlined by javac. There's no way for sure the getters are in the same order as the fields, and there's no point in mapping them anyway.

Reply:
Do the fields have the same values? If they cannot identify by value (like a few private static final fields in piglin/hoglin) then i'd leave them unmapped.

Reply:
Three of them match three of the getters, but `field_13221` ("`minY`") does not match the `getMinY` return value.